test/test_main.py

```diff
@@ -287,43 +279,47 @@ def _assess_setup_hooks(event_spec):
 
             for event_hook in event_hooks:
                 self.assertTrue(os.path.exists(event_hook))
-                self.assertEqual(os.readlink(event_hook), 'install')
-                self.assertEqual(os.readlink('hooks/install'),
-                                 self.CHARM_PY_RELPATH)
-
+                self.assertEqual(os.readlink(event_hook), op.main.CHARM_CODE_FILE)
```

Comment:
Same here and in other places in tests.

Reply:
See https://github.com/canonical/operator/pull/63/files#r352704416